hw/ip/flash_ctrl/dv/xfile

```diff
@@ -0,0 +1 @@
+SCOPE tb.dut F
```

Comment:
Seems this file was added by mistake?

Reply:
No, Xcelium does not work without following switch:
--build-opts "-xfile /projects/google/goo_opentitan/nikolam/opentitan/hw/ip/flash_ctrl/dv/xfile".
VCS works normal, but this switch is needed in order to override integrated XPROP in Xcelium. Otherwise Xcelium stays in infinite loop.